dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/user/hibernate/HibernateUserStore.java

```diff
@@ -480,6 +481,23 @@ private Map<String, Optional<Locale>> toLocaleMap(String hql, Date from, Date to
                 (Object[] columns) -> Optional.ofNullable((Locale) columns[1])));
   }
 
+  @Override
+  public Map<String, String> getUserGroupUserEmailsByUsername(String userGroupId) {
```

Comment:
we should create a dedicated store test covering this native query

Reply:
Test added, thanks for the reminder